modules/data/config.py

```diff
@@ -180,6 +180,7 @@ def __read_env_settings(self):
         new_vars = {}
         self.settings['test'] = self.settings['test'] if self.settings.get('test', False) else {}
         self.settings['meme'] = self.settings['meme'] if self.settings.get('meme', False) else {}
+        self.settings['debug'] = self.settings['debug'] if self.settings.get('debug', False) else {}
```

Comment:
```suggestion
        self.settings['debug'] = self.settings.get('debug', {})
```

this should work as well and it's cleaner (I did not test it), if you change it here, change also the other two lines please